packages/go_router/lib/src/information_provider.dart

```diff
@@ -274,11 +278,4 @@ class GoRouteInformationProvider extends RouteInformationProvider
     _platformReportsNewRouteInformation(routeInformation);
     return SynchronousFuture<bool>(true);
   }
-
```

Comment:
why remove this?

Reply:
See [here](https://github.com/flutter/packages/pull/5742#issuecomment-1871364697).

[`didPushRoute` is now deprecated, and `didPushRouteInformation` should be used instead](https://github.com/flutter/flutter/blob/4721f96c3042281a80472047a4de06cd17755350/packages/flutter/lib/src/widgets/binding.dart#L84-L88).